catalog/models.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.urls import reverse
+
+
+class DishType(models.Model):
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return self.name
+
+
+class Cook(AbstractUser):
+    years_of_experience = models.IntegerField(default=0)
+
+    class Meta:
+        verbose_name = "Cook"
+        verbose_name_plural = "Cooks"
+        ordering = ["id"]
+
+    def __str__(self):
+        return f"{self.username} ({self.first_name} {self.last_name})"
+
+    def get_absolute_url(self):
+        return reverse("catalog:cook-detail", kwargs={"pk": self.pk})
+
+
+class Dish(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    price = models.DecimalField(decimal_places=2, max_digits=10)
+    dish_type = models.ForeignKey(DishType, on_delete=models.CASCADE)
+    cooks = models.ManyToManyField(Cook, related_name="dishes")
+
```

Comment:
Dobrą praktyką jest również dodawanie pól odpowiedzialnych za tracking utworzenia i zmian
```python
created_at = models.DateTimeField(auto_now_add=True)
updated_at = models.DateTimeField(auto_now=True
```
Możesz w ten sposób śledzić kiedy dany obiekt został utworzony oraz kiedy został aktualizowany. Oczywiście to tylko sugestia na przyszłość